src/Workspaces/MSBuild/Test/TestFiles/Resources.cs

```diff
@@ -71,6 +71,11 @@ public static class SolutionFilters
             public static string Invalid => GetText("SolutionFilters.InvalidSolutionFilter.slnf");
             public static string CSharp => GetText("SolutionFilters.CSharpSolutionFilter.slnf");
         }
+        public static class XmlSolutions
```

Comment:
nit: Newline between classes
```suggestion

        public static class XmlSolutions
```